### PR TITLE
Fix #3073

### LINKED ISSUE #3073: Aegean Airlines search field not working
The search fields of Aegean Airlines (http://en.aegeanair.com/) are not working, thus not giving any suggestions for departure airport or destination airport. Only after i disable https-everywhere, it works. Please help.

### PR DESCRIPTION
**Pull request by galeksandrp**
and #3392

### CODE REVIEW

**Comment by J0WI** [comment]
Sadly this is a wildcard host.
So you need test urls for at least three hosts and a further test url for the exclusion.
I generally recommend to avoid wildcards and use a list of targets (this also makes the three test urls obsolete).

**Comment by J0WI** [comment]
WFM:
alphabankbonusredemptionel.aegeanair.com
alphabankbonusredemptionen.aegeanair.com
extranet.aegeanair.com

Works for some resources:
de.about.aegeanair.com
it.about.aegeanair.com
fr.about.aegeanair.com

Mismatch:
eastpak.aegeanair.com
hotel.aegeanair.com
flights.aegeanair.com

**Comment by galeksandrp** [comment]
Added.

File: src/chrome/content/rules/Greek-travel.xml [hunk 1]
```diff
@@ -1,10 +1,43 @@
...
+	<target host="en.about.aegeanair.com" />
+	<target host="alphabankbonusredemptionen.aegeanair.com" />
+	<target host="alphabankbonusredemptionel.aegeanair.com" />
+	<target host="el.about.aegeanair.com" />
+	<target host="it.about.aegeanair.com" />
+	<target host="de.about.aegeanair.com" />
+	<target host="fr.about.aegeanair.com" />
 
+	<!-- Fix #3073 and #3392 -->
+	<exclusion pattern="^http://(?:en|el|de|it|fr|es|ru|(?:en|el|de|it|fr).about)\.aegeanair\.com/(?:sys/flights/|Css/fonts/|$|\?|#)" />
```
Comment (J0WI) [inline src/chrome/content/rules/Greek-travel.xml]: code style: avoid non-capturing group

File: src/chrome/content/rules/Greek-travel.xml [hunk 2]
```diff
@@ -1,10 +1,43 @@
...
+	<test url="http://en.aegeanair.com/Css/fonts/Roboto-Bold/Roboto-Bold.ttf" />
+	<test url="http://el.aegeanair.com/Css/fonts/Roboto-Bold/Roboto-Bold.ttf" />
+	<test url="http://de.aegeanair.com/Css/fonts/Roboto-Bold/Roboto-Bold.ttf" />
+	<test url="http://it.aegeanair.com/Css/fonts/Roboto-Bold/Roboto-Bold.ttf" />
+	<test url="http://fr.aegeanair.com/Css/fonts/Roboto-Bold/Roboto-Bold.ttf" />
+  
 	<securecookie host="^(?:.*\.)?aegeanair\.com$" name=".*"/>
 
-  <rule from="^http://(\w+\.)?aegeanair\.com/" to="https://$1aegeanair.com/"/>
+  <rule from="^http:"
```
Comment (J0WI) [inline src/chrome/content/rules/Greek-travel.xml]: code style: use tabs

File: src/chrome/content/rules/Greek-travel.xml (hunk 1 above)
Comment (J0WI) [inline src/chrome/content/rules/Greek-travel.xml]: This also needs more test urls to cover all cases.

**Comment by galeksandrp** [comment]
Done.

**Comment by J0WI** [comment]
Note that you can still use capturing groups:

> Prefer capturing groups `(www\.)?` over non-capturing `(?:www\.)?`. 

Rule LGTM.

**Pull request merged**